pal/pacer/pacer-web.ts

```diff
@@ -96,32 +97,41 @@ export class Pacer {
         this._stHandle = this._stTime(updateCallback);
 
         this._isPlaying = true;
+        this._frameCount = 0;
     }
 
     stop (): void {
         if (!this._isPlaying) return;
         this._ctTime(this._stHandle);
         this._stHandle = 0;
         this._isPlaying = false;
+        this._frameCount = 0;
     }
 
     _handleRAF = () => {
-        if (performance.now() - this._start < this._delay) {
+        const elapseTime = performance.now() - this._startTime;
+        const elapseFrame = Math.floor(elapseTime / this._frameTime);
+        if (elapseFrame < this._frameCount) {
```

Comment:
Here the elapseFrame is only used to determine if the number of frames has changed. Then why not just
if (this._frameTime * this._frameCount > elapseTime) {

Reply:
It is also correct. Diffenert code means different perspective of understanding.